### PR TITLE
Unify error message casing

### PR DESCRIPTION
**Pull request by romac**
## Description

Start error message with a lower case letter everywhere.

______

### PR author checklist:
- [ ] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules). 
- [ ] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

**Review by adizere — APPROVED** [review]
Love it! Left two questions, feel free to merge.

I will do a bit of noise by tagging the whole core team here, as an unofficial signal that we're adopting lower-cases in log messages throughout the codebase. @hu55a1n1 @mzabaluev @seanchen1991  @soareschen.

File: relayer/src/channel.rs [hunk 1]
```diff
@@ -866,7 +866,7 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> Channel<ChainA, ChainB> {
         let counterparty =
             Counterparty::new(self.src_port_id().clone(), self.src_channel_id().cloned());
 
-        let version = version::default_by_port(self.dst_port_id())?;
+        let version = version::default_by_port(self.dst_port_id()).unwrap_or_default();
```
[the remark this answers is not in the document]
Comment (adizere) [reply relayer/src/channel.rs:869]: Feel free to ignore this comment. I just saw #1840.

File: relayer/src/error.rs [hunk 2]
```diff
@@ -50,14 +50,14 @@ define_error! {
...
-            |e| { format!("ABCI query returns error: {:?}", e.query) },
+            |e| { format!("ABCI query returned an error: {:?}", e.query) },
 
         CheckTx
             {
                 detail: SdkError,
                 tx: TxResult
             }
-            |e| { format!("CheckTX Commit returns error: {0}. RawResult: {1:?}", e.detail, e.tx) },
+            |e| { format!("CheckTX Commit returned an error: {0}, raw result: {1:?}", e.detail, e.tx) },
```
Comment (mzabaluev) [inline relayer/src/error.rs:60]: The accurate capitalization should be `CheckTx`?

Comment (romac) [inline relayer/src/error.rs:60]:
```suggestion
            |e| { format!("CheckTx commit returned an error: {0}, raw result: {1:?}", e.detail, e.tx) },
```

File: relayer/src/channel.rs (hunk 1 above)
Comment (romac) [inline relayer/src/channel.rs:869]:
```suggestion
        let version = version::default_by_port(self.dst_port_id())?;
```

**Review by mzabaluev — APPROVED** [review]

File: proto-compiler/src/cmd/compile.rs [hunk 3]
```diff
@@ -119,7 +123,12 @@ impl CompileCmd {
...
 
-    fn compile_sdk_protos(sdk_dir: &Path, out_dir: &Path, ibc_dep: Option<PathBuf>, build_tonic: bool) {
+    fn compile_sdk_protos(
+        sdk_dir: &Path,
+        out_dir: &Path,
+        ibc_dep: Option<PathBuf>,
+        build_tonic: bool,
+    ) {
         println!(
             "[info ] Compiling Cosmos-SDK .proto files to Rust into '{}'...",
```
[the remark this answers is not in the document]
Comment (romac) [reply proto-compiler/src/cmd/compile.rs:133]: This is only in the proto compiler where it doesn't really matter. Let's leave it as is for now.

**Pull request merged**